### PR TITLE
Update some cases

### PR DESCRIPTION
**Pull request by tzhou5**
1. Add cases: OCP-22101, OCP-21090, OCP-27839
2. Add checkpoint in OCP-21209
3. Update links in install cases, OCP-21338,OCP-24068,OCP-24069,OCP-24071,OCP-25176,OCP-24072,OCP-24070,OCP-26297,OCP-25174,OCP-27839,OCP-25175,OCP-26380
4. Update some button into lower case,OCP-23574,OCP-21339,OCP-25353
5. Update user login method into 'as an *** user'

Here are Jenkins log about install page cases: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/71651/console
And other cases run by Jenkins:https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/71669/console

@yasun1 @xueli181114 @yuwang-RH Please help review cases, thanks.

### CODE REVIEW

**Comment by yasun1** [comment]
@yuwang-RH  @xueli181114 Please help review too.  When I review the rules, I found some dynamical values are hardcoded. It is not a good. We need pass them as variables to avoid updating rules frequently in future(updating case should be more acceptable).  And for the checking related to personal information, etc. username, advise do them manually or in an individual automation case so that reduce the affection by private info.

**Comment by tzhou5** [comment]
Update cases related with subscription settings
Fix username hard code issue
Simplify some rules, make it more readable

Here are Jenkins log for updated cases: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/74183/console

File: lib/rules/web/ocm_console/cluster_list_page.xyaml [hunk 1]
```diff
@@ -341,6 +358,212 @@ go_to_disconnected_cluster_register_page:
...
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Actual Nodes']
+        - selector:
+            xpath: //dl[@class='cluster-details-item left']/dt[text()='Network']
+expand_actions_behind_open_console_button:
+    element:
+        selector:
+            xpath: //span[@class='pf-c-dropdown__toggle-text' and text()='Actions']/../../button
+        op: click
+check_ocp_actions_behind_open_console_button:
```
Comment (xueli181114) [inline lib/rules/web/ocm_console/cluster_list_page.xyaml]: All of these actions should not be defined in cluster_list_page.xyaml. They should be defined in cluster_detail.xyaml

File: features/step_definitions/ocm.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+require 'webauto/webconsole_executor'
```
Comment (xueli181114) [inline features/step_definitions/ocm.rb]: Is this what we indeed need to modify?

**Review by xueli181114 — CHANGES_REQUESTED** [review]

File: features/step_definitions/ocm.rb (hunk 2 above)
Comment (tzhou5) [reply features/step_definitions/ocm.rb]: This is added in yuwan's PR in https://github.com/xueli181114/verification-tests/pull/24/files.
Without this line, our code cannot run in Jenkins.

File: lib/rules/web/ocm_console/cluster_list_page.xyaml (hunk 1 above)
Comment (tzhou5) [reply lib/rules/web/ocm_console/cluster_list_page.xyaml]: Updated

**Comment by tzhou5** [comment]
Also add tooltip checkpoint in OCP-23574.
Here is Jenkins log: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/75535/console

**Pull request merged**